docs/manuals/core/config/job-config.md

```diff
@@ -114,4 +114,67 @@ For example, the following configuration will allow Taipy to run up to eight `Jo
     max_nb_of_workers = "8:int"
     ```
 
+# Cluster mode
+
+!!! warning "Available in Taipy Enterprise edition"
+
+    This section is relevant only to the Enterprise edition of Taipy.
+
+With the *cluster* mode, a `Job^` runs in its own environment (a worker environment), which can be
+local or on a remote machine, in an asynchronous manner. At the submission, the job is queued in a
+RabbitMQ queue. When there is a worker available, the job is dequeued and sent to the worker
+environment to be executed.
+
+Note that with the *cluster* mode, the submit method is not blocking and returns after the job is queued.
+It means the submit method can return before the job finishes or even before it is dequeued. Please refer to
+the [submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-sequence-or-task) section
+on how to submit jobs.
+
+You can configure the *cluster* mode with the following config:
+
+=== "Python configuration"
+
+    ```python linenums="1"
+    from taipy import Config
+
+    Config.configure_job_executions(mode="cluster")
+    ```
+
+=== "TOML configuration"
+
+    ```python linenums="1"
+    from taipy import Config
+
+    Config.load("config.toml")
+    ```
+
+    ```toml linenums="1" title="config.toml"
+    [JOB]
+    mode = "cluster"
+    ```
+
+To set up a worker environment, some requirements must be met:
+
+- The worker environment must have access to the same Taipy application as the main environment.
+- The worker environment must have the same Python environment as the main environment.
+- The worker environment must have access to the same RabbitMQ server as the main environment.
+- The worker environment must have access to the data of the data nodes as the main environment.
+  This includes:
+  - The shared drive(s) containing the data for file-based data nodes.
+  - The database server(s) containing the data for database-based data nodes.
```

Comment:
Reformulation. What do you think?

```suggestion
- The main application and the workers must have the same Python environment.
- The main application and the workers must have access to the same RabbitMQ server.
- The main application must be configured to use a Mongo repository, and the workers must have access to it as well.
- The main application and the workers must have access to the data referenced by the the data nodes. This includes:
    - The shared drive(s) containing the data for file-based data nodes.
    - The database server(s) containing the data for database-based data nodes.
```

Reply:
The idea behind my proposal is to explain for the user how to set up 1 worker. I call each worker as "worker environment" because the worker can be remote or local, it just needs to be a separate environment.

About the "Mongo repository" requirement, how about "must have access to the entities"?
Mongo is nice to have, but not strictly.

Reply:
1. **About worker environment:** For me, the whole feature makes sense only on remote machines. It does not make sense to me to run a local worker. We have the standalone mode for that. This is only useful for us as contributors, but not really for our users. What do you think?

2. **About mongo:** I thought a `mongo` repository was mandatory to run in `cluster` mode. Is it possible to run a remote worker on a filesystem repository, for instance? If yes, I agree with your proposal.